headapps/MvpSite/MvpSite.Rendering/Views/Shared/Components/Profile/Default.cshtml

```diff
@@ -44,11 +44,14 @@
                 <partial name="Components/Profile/_Mentor" model="Model" />
             </div>
         }
-        <div class="container timeline ">
-            <div class="row inner mb-5">
-            <h3 asp-for="TimelineLabel" class="col-12"></h3>
-            <partial name="Components/Profile/_Timeline" model="Model.GetTimelineEvents()" />
-        </div>
-        </div>
+        @if (Model.GetTimelineEvents()?.Any() == true)
```

Comment:
I dont like the comparison to true here as it feels like it's hiding the fact that the code is stating that `GetTimelineEvents()` could be null.

Can we change this use a Null-Coalescing operator instead?

e.g.

`@if (Model.GetTimelineEvents()?.Any() ?? false)`